src/celeritas/em/xs/WentzelHelper.hh

```diff
@@ -237,14 +247,13 @@ CELER_FUNCTION real_type WentzelHelper::calc_screening_coefficient(
    C_{TF} = \frac{1}{2} \left(\frac{3\pi}{4}\right)^{2/3}.
  * \f]
  */
-CELER_CONSTEXPR_FUNCTION real_type WentzelHelper::screen_r_sq_elec() const
+CELER_CONSTEXPR_FUNCTION auto WentzelHelper::screen_r_sq_elec() -> MomentumSq
```

Comment:
Is the removal of the `const` intentional?

Reply:
Yep, it's a static function now since it doesn't use any member data: just a constexpr helper function.